<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>892</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#container {
	background-color: #323a47;;
	border-radius: 5px;
	color:white; 
} 

QPushButton { 
	padding: 4px;
	border: 1px solid #30333b; 
	border-radius: 5px;
	color:white; 
	background-color: #555555; 
	min-width: 100px; 
} 

QPushButton::hover {
	background-color: rgb(107, 108, 108);
}

QPushButton::disabled {
	color: grey 
}


QLabel {
	color: white; 
} 

QLineEdit {
	background-color: #363f4d; 
	border: 1px solid #30333b; 
	color:white; 

}

QComboBox {
	background-color: #363f4d;
	color:white; 
}

QComboBox::drop-down {
	background-color: #363f4d;
}


QListView{
	background-color : #555555;
	padding: 0px; 
	margin: 0px; 
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="container" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget_34" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="label_53">
           <property name="text">
            <string>Job Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="jobNum"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_36" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QLabel" name="label_54">
           <property name="text">
            <string>Sample Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sampleNum"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_37" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_28">
         <item>
          <widget class="QLabel" name="label_56">
           <property name="text">
            <string>Tests Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="testsNameCombo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_39" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_29">
         <item>
          <widget class="QLabel" name="label_57">
           <property name="text">
            <string>Tests Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="testsVal"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_44" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_33">
         <item>
          <widget class="QLabel" name="label_61">
           <property name="text">
            <string>Standard Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="standard"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_47" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_35">
         <item>
          <widget class="QLabel" name="label_64">
           <property name="text">
            <string>Unit Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="unitValCombo"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_20" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_63">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cancelBtn">
           <property name="text">
            <string>Cancel Changes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="text">
            <string>Save Changes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
